packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -16,32 +16,32 @@ For information about the element that opens the dropdown container, read the ar
 
 ## Options
 
-- **Header:** Add a header for the dropdown container. This provides a better understanding for the user what items are found in the dropdown.
+- **Header:** Add a header for the dropdown container. This helps the user to better understand which elements are in the dropdown.
 - **Quick Actions:** Add a quick action bar. We typically use quick actions to access common functions such as cut, copy and paste.
-- **Checked:** Mark selected items in the dropdown with a check mark.We typically use check marks if an item can be activated (e.g. for the dropdown "Sort by:" selection: Name, Change date, Create date).
+- **Checked:** Mark selected items in the dropdown with a check mark. We typically use check marks when an item can be activated (e.g. for the dropdown "Sort by:" selection: Name, Change date, Create date).
 - **Submenu:** Add a submenu for a multi-level dropdown. We typically use a submenu when there is a long item list and a varied categorization within the items.
-- **Separator:** Add a separator to visually divide items from each other. 
-- **Icon:** Icons can also be displayed with item labels. The symbols can also be displayed together with the item descriptions. We typically use a symbol in addition to the label if it has a recognizable benefit.
+- **Separator:** Add a separator to visually divide items from each other. We normally use a separator to isolate individual elements from a cohesive item list.
+- **Icon:** Icons can also be displayed with item labels. We typically use an icon in addition to the label when it has a recognizable benefit.
 - **Label:** Set a label for the dropdown item. We typically use short labels including verbs.
-- **Anchor:** An anchor defines where the dropdown is opened. If no anchor is defined the trigger element is used as the anchor.
-- **Trigger:** The trigger defines which element opens the dropdown. A trigger should also be defined for a dropdown submenu. Typically the Button is the trigger element.
+- **Anchor:** An anchor defines where the dropdown is opened. When no anchor is defined the trigger element is used as the anchor.
```

Comment:
since you refer to the trigger element: Maybe swap those two, then the reader already knows about the trigger element :)

Reply:
tbh I personally don't fully understand the difference between the two options 🤔 maybe we can have a chat about this :)